### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by Elfin84** [WestSide44/Guide-Fuel-Beta-4#2] [Q&A] [answered]
Hello can you please help me with this error

( Error: Provider error: Response errors; Unknown field “resourcesToSpend” on type

“Query”. Did you mean “coinsToSpend”?; Unknown field “status” on type “Coin” Unknown field “messageId” on type “Message”.; Unknown field “status” on type

"Me

ssage".)
I keep getting the error message when I tried to deploy contract with beta-4 wallet

**Comment by W3stS1de** [comment, marked as answer]
This error is most likely due to a version mismatch, meaning either your app, wallet, or the blockchain itself is out of date
